.buildkite/pipeline.yml

```diff
@@ -17,6 +17,8 @@ steps:
         image-repository: 027047743804.dkr.ecr.us-east-2.amazonaws.com/uber
     agents:
       queue: builders
+    concurrency: 1
+    concurrency_group: 'e2e-and-vrt-test-server'
```

Comment:
@gergelyke I tried this out as a solution to the port clash race condition- the thinking here being that if we remove the server build race condition if this to being built one at a time.

Reply:
i think the issue is not with the builders, but when it runs

Reply:
yeah this doesn't work